src/components/BookDropDown/BookDropDown.md

```diff
@@ -0,0 +1,59 @@
+### Context API example
+
+```js
+import { useState } from 'react';
+import SoureList from '../SourceList';
+import BookDropDown from './BookDropDown';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import ProjectSelect from '../ProjectSelect';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    padding: theme.spacing(2),
+    margin: 'auto',
+    maxWidth: 600,
+    maxHeight: 400,
+  },
+}));
+
+const classes = useStyles();
+const [bookChapter, setBookChapter] = useState({ book: 'gen', chapter: 1 });
+const [source, setSource] = useState([]);
+const [project, setProject] = useState([]);
+
+<>
+  <Grid container spacing={3} className={classes.gridContainer}>
+    <Grid item xs={6}>
+      <ProjectSelect value={project} onChange={setProject} />
+      <Button
```

Comment:
We can remove the clear button as there is clear in the select project dropdown
Currnetly it is giving some error on clicking x
We can do it after the demo later